### ISSUE TITLE
PharoObjectModel - edit or delete references to 'Color>>initializeNames'

### ISSUE THREAD

**Issue by dmitrizagidulin** [SquareBracketAssociates/UpdatedPharoByExample#107]
The end of the 'Class variables' section used the `ColorNames` class variable as an illustration of the shared nature of class variables. In Pharo 4, `ColorNames` has been replaced by `ColorRegistry`. 

Which is fine, I changed the example to refer to `ColorRegistry`. But the last paragraph (currently commented out, see [lines 1072-1074](https://github.com/SquareBracketAssociates/UpdatedPharoByExample/blob/1026f60bbe982fabc3245a84c89f04ccb61c32fb/PharoObjectModel/PharoObjectModel.pier#L1072-L1074)), mentioned that `ColorNames` is accessed from the *instances* of `Color`. Whereas the new `ColorRegistry` is primarily used by class methods in Color.

My question is, should we:
1) Just omit that last paragraph, since we made our overall point without it
2) Use some other class variable of Color
3) Use a class other than Color (since in 4, it doesn't have any subclasses, for example) and another class variable?

/cc @Ducasse

**Comment by nicolaihess** [comment]
Yes the method Color>>name not even accesses this variable from the instance side.
Mybe class Cursor is a better example for class variables.
